src/08-tasks/tasks.apib

```diff
@@ -12,6 +12,7 @@ Get a list of tasks.
             + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional, nullable) - Returns tasks that are assigned to this user or to a team to which this user belongs. When passing `null`, it returns tasks that are unassigned.
             + milestone_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
             + completed: true (boolean, optional)
+            + scheduled: true (boolean, optional)
```

Comment:
Q: should we make the value of `scheduled` explicit in the task data in the info and list endpoint?

Reply:
Are you suggesting that the info and list requests would return something like: 
`scheduled: true`?

Reply:
> Are you suggesting that the info and list requests would return something like:
> `scheduled: true`?

Indeed. I would also go that route if the implementation isn't too complicated.

Reply:
PP: have the `scheduled` property already in the list and info endpoint return data. This will also help us with testing.

Reply:
It's actually quite a lot of work to return the scheduled property. It can be done for sure but it involves joining different tables. Which I find a bit difficult. 
I can do it but I think I will have to ask Pieter if this has priority for my time. 

Due to merge conflicts on this branch I will close this Pr and open a new one.